Util/BuildTools/BuildPythonAPI.bat

```diff
@@ -153,4 +153,4 @@ rem ============================================================================
 
 :bad_exit
     endlocal
-    exit /b %errorlevel%
+    exit /b %errorlevel%
```

Comment:
Something trivial. But, It seems that you deleted the empty line at the end of the file.

```suggestion
    exit /b %errorlevel%
    
```

Reply:
Hm, it seems I did. IDK if the whole repo attempts to maintain posix compliance but nonetheless unnecessary changes are bad so I will gladly accept this